docs/src/pages/components/authenticator/advanced/current-route.angular.mdx

```diff
@@ -26,3 +26,23 @@ You can use `AuthenticatorService` to access `route` string that represents the
   <amplify-authenticator></amplify-authenticator>
 </ng-container>
 ```
+
+#### Authentication Check
+
+If you just need to check if you're authenticated or not, you can use the more straight forward `AuthenticatorService` to access the `authCheck` string. The `authCheck` string can represent the following states:
+
+- `pending`
+- `authenticated`
+- `unAuthenticated`
```

Comment:
I would lowercase this to `unauthenticated`

Reply:
ok

Reply:
dd3e3f0d9511a8d667d8223306538db8f0a84f96